internal/cmd/cmd_test.go

```diff
@@ -86,6 +87,29 @@ func newTestClientConfig(srv *httptest.Server, user api.User) ClientConfig {
 	}
 }
 
+func newTestClientConfigForServer(srv *server.Server, user api.User, acKey string) ClientConfig {
+	if user.Name == "" {
+		user.Name = "testuser@example.com"
+	}
+	if user.ID == 0 {
+		user.ID = uid.New()
+	}
+	return ClientConfig{
+		ClientConfigVersion: clientConfigVersion,
+		Hosts: []ClientHostConfig{
+			{
+				UserID:             user.ID,
+				Name:               user.Name,
+				Host:               srv.Addrs.HTTPS.String(),
+				TrustedCertificate: string(srv.Options().TLS.Certificate),
```

Comment:
why not just use `srv.Options.TLS.Certificate`?

Reply:
`srv.options` is not exported, so we can't access it from here (different package).